tests/topotests/bgp_gr_notification/test_bgp_gr_notification.py

```diff
@@ -192,19 +192,23 @@ def _bgp_clear_r1_and_shutdown():
     _, result = topotest.run_and_expect(test_func, None, count=60, wait=0.5)
     assert result is None, "Failed to see BGP convergence on R2"
 
+    import time
+
+    time.sleep(10)
```

Comment:
This MUST be fixed, no `sleep` in tests please.

Reply:
Removed it